dali/pipeline/data/tensor_test.cc

```diff
@@ -344,14 +343,14 @@ TYPED_TEST(TensorTest, TestResize) {
   Tensor<TypeParam> tensor;
 
   // Get shape
-  vector<Index> shape = this->GetRandShape();
+  auto shape = this->GetRandShape();
   tensor.Resize(shape);
 
   // Verify the settings
   ASSERT_NE(tensor.template mutable_data<float>(), nullptr);
   ASSERT_EQ(tensor.size(), volume(shape));
   ASSERT_EQ(tensor.ndim(), shape.size());
-  for (size_t i = 0; i < shape.size(); ++i) {
+  for (int i = 0; i < shape.size(); ++i) {
```

Comment:
Do we know why `shape.size()` is signed?

Reply:
We decide to do that, to be compatible with the `ntensor` and stuff we already have (the operate with signed int64_t/Index).